### PR TITLE
Finished Rotating Marquee

### PR DESCRIPTION
**Pull request by pantp**
I have a basic rotating marquee working. Since this is a small feature, I don't want to dedicate too much time on it. In the future, we could add a stationary banner.

### CODE REVIEW

File: home/templates/home/home_page.html [hunk 1]
```diff
@@ -33,7 +33,15 @@
 <div class="container">
     {% for block in page.featured_articles %}
         <div class="row mb-md-4">
-            {% if block.block_type == "one_column" %}
+            {% if block.block_type == "marquee_banner" %}
+                {% if block.value.banner_type == "moves" %}
+                    <div class="w-100">
+                        <marquee onmouseover="this.stop();" onmouseout="this.start();">
```
Comment (jlyon1) [inline home/templates/home/home_page.html]: We should probably break the calls to `this.` into it's own function, this should work, but I don't like the style

**Comment by jlyon1** [comment]
Also you need to run 
`black core/models.py`
`black home/models.py`
to fix the formatting issues.

File: home/templates/home/home_page.html (hunk 1 above)
Comment (pantp) [reply home/templates/home/home_page.html]: Function for stopping on the marquee on hover? Using the onmouseover tag seems like the simplest way to do it.

**Comment by pantp** [comment]
I did run black locally on those files a few weeks ago and wasn't getting this error. I can try again and push.

**Pull request merged**